src/data/ACTIONS/MNK.js

```diff
@@ -0,0 +1,135 @@
+export default {
+	// -----
+	// Player GCDs
+	// -----
+
+	ROCKBREAKER: {
+		id: 70,
+		name: 'Rockbreaker',
+		icon: 'https://secure.xivdb.com/img/game/002000/002529.png',
+		onGcd: true,
+	},
+
+	ONE_ILM_PUNCH: {
+		id: 72,
+		name: 'One-Ilm Punch',
+		icon: 'https://secure.xivdb.com/img/game/000000/000214.png',
+		onGcd: true,
+	},
+
+	DRAGON_KICK: {
+		id: 74,
+		name: 'Dragon Kick',
+		icon: 'https://secure.xivdb.com/img/game/002000/002528.png',
+		onGcd: true,
+	},
+
+	FORM_SHIFT: {
+		id: 4262,
+		name: 'Form Shift',
+		icon: 'https://secure.xivdb.com/img/game/002000/002536.png',
+		onGcd: true,
+	},
+
+	// -----
+	// Player OGCDs
+	// -----
+
+	SHOULDER_TACKLE: {
+		id: 71,
+		name: 'Shoulder Tackle',
+		icon: 'https://secure.xivdb.com/img/game/002000/002526.png',
+		cooldown: 30,
+	},
+
+	FISTS_OF_FIRE: {
+		id: 63,
+		name: 'Fists of Fire',
+		icon: 'https://secure.xivdb.com/img/game/000000/000205.png',
+		cooldown: 3,
+	},
+
+	THE_FORBIDDEN_CHAKRA: {
+		id: 3547,
+		name: 'The Forbidden Chakra',
+		icon: 'https://secure.xivdb.com/img/game/002000/002535.png',
+		cooldown: 5,
+	},
+
+	MEDITATION: {
```

Comment:
Shouldn't this technically count as `onGcd: true`? It's a short CD, but it still hits all your regular GCD abilities.

Reply:
You're right, after a chat in Discord it's moving to GCD and going to implement a module to deal with the different timer on it, similar to SMN/RDM.